src/service/store/businessStoreService.ts

```diff
@@ -84,5 +84,14 @@ const deleteStore = async (storeId: number) => {
     }
 }
 
+const getStoreDetails = async (storeId: number) => {
+    try {
+        const store = await businessStoreRepo.getStoreDetails(storeId)
+        return store
+    }
+    catch (e) {
+        throw e
```

Comment:
I have another way of handling errors but it needs more testing, so I'll just leave this comment to remind me later